commands/utilities/clean.js

```diff
@@ -0,0 +1,38 @@
+const { SlashCommandBuilder, Message } = require("discord.js");
+const { blacklist } = require('../../utility/blacklist');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName("clean")
+        .setDescription("Deletes a specified number of stickers from a specific user within 100 latest messages.")
+        .addStringOption(option =>
+            option.setName('user_id')
+            .setDescription('ID of user whose messages you want to delete')
+            .setRequired(true)
+        )
+        .addIntegerOption(option =>
+            option.setName('max_count')
+            .setDescription('Amount of messages you want to delete.')
+            .setRequired(true)
+        ),
+    async execute(interaction) {
+        const user_id = interaction.options.getString('user_id');
+        const max_count = interaction.options.getInteger('max_count');
+        
+        await interaction.channel.messages.fetch({ limit: 100 })
+            .then(messages => {
+                let msgDeleted = 0;
+                for (const message of messages) {
+                    // each message is an array, so message[1] returns the Message object containing author, id, and other properties
+                    if (message[1].stickers.size > 0 && message[1].author.id === user_id && msgDeleted < max_count && blacklist.has(message[1].id) === false) {
+                        // message[1] is the actual message object, not message. therefore, the delete function is called on message[1]
+                        message[1].delete();
+                        msgDeleted++;
+                    }
```

Comment:
If we reach our goal of deleted messages, wouldn't this keep iterating over? Can we add a break statement once we reach the goal?